### ISSUE TITLE
Feature Request: Support XMPP Offline Messages (XEP-13)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#144]
```
http://www.igniterealtime.org/builds/smack/docs/3.1.0/javadoc/org/jivesoftware/s
mackx/OfflineMessageManager.html

If we detect a server with support, we should enable this and query for 
messages that where send while gtalksms was offline.


```

Original issue reported on code.google.com by `fschm...@gmail.com` on 17 Apr 2011 at 8:34

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 17 Apr 2011 at 10:50
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.2
```

Original comment by `Florent....@gmail.com` on 24 Apr 2011 at 1:16
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
